integration/detach_disk/detach_disk_test.go

```diff
@@ -184,7 +186,7 @@ var _ = Describe("BOSH Director Level Integration for detach_disk", func() {
 		})
 	})
 
-	Context("detach_disk in SoftLayer with valid virtual guest id(with no multipath installed) and disk id", func() {
+	/*	Context("detach_disk in SoftLayer with valid virtual guest id(with no multipath installed) and disk id", func() {
 		BeforeEach(func() {
```

Comment:
Same again. Commenting code like this just to get things working is recipe for future failure. This is production. Doing things like that is only when code is throw away...  The simple problem is in future, no one will remember why this was commented (removed) and/or why no tests. We are not in this to get code working for one instance in time. WE need to create quality code that lasts a long time.